packages/terra-notification-dialog/CHANGELOG.md

```diff
@@ -3,6 +3,16 @@ Changelog
 
 Unreleased
 ----------
+### Breaking Changes
```

Comment:
This might have already been discussed, but can we keep these props and just map them to the new props if they're provided? Mark them as deprecated in the props table with suggestions to use the other props?

Reply:
Taken care of here 
https://github.com/cerner/terra-framework/pull/899/commits/67555271fd37cfa811ddb2cf31151f65149929b1#diff-cdc6379ea8cdd76265a948289e4bdab4R10
https://github.com/cerner/terra-framework/pull/899/commits/42e0b275d5836df7a19ea7618586e1cb8dd730fd